db/permissions_roles.yml

```diff
@@ -89,6 +89,8 @@ Stock fulfilment: &stock_fulfilment
 - can_manage_stocktake_revisions
 - can_manage_package_messages
 - can_read_users
+- can_access_organisation_types
```

Comment:
I don't think we need a permission for viewing organisation_types. Anyone should be allowed to view them

Reply:
I have removed it